packages/server/graphql/schema.js

```diff
@@ -3,31 +3,75 @@ const {
   GraphQLObjectType,
   GraphQLID,
   GraphQLList,
-  GraphQLNonNull
+  GraphQLNonNull,
+  GraphQLString
 } = require("graphql");
-const fromGlobalId = require("graphql-relay").fromGlobalId;
-const productGraphQLType = require("./productType");
-const Product = require("../models/Product");
+const { fromGlobalId, mutationWithClientMutationId } = require("graphql-relay");
+const eventGraphQLType = require("./eventType");
+const Event = require("../models/Event");
 
 const Query = new GraphQLObjectType({
   name: "Query",
   fields: {
-    product: {
-      type: productGraphQLType,
+    event: {
+      type: eventGraphQLType,
       args: { id: { type: GraphQLNonNull(GraphQLID) } },
       resolve(parent, args) {
-        return Product.findById(fromGlobalId(args.id).id);
+        return Event.findById(fromGlobalId(args.id).id);
       }
     },
-    products: {
-      type: GraphQLList(productGraphQLType),
+    events: {
+      type: GraphQLList(eventGraphQLType),
       resolve() {
-        return Product.find().lean();
+        return Event.find();
       }
     }
   }
 });
 
+const EventCreate = mutationWithClientMutationId({
```

Comment:
move this to another file

Reply:
just did with the new subscription branch